.changeset/config.json

```diff
@@ -0,0 +1,14 @@
+{
+    "$schema": "https://unpkg.com/@changesets/config@2.3.0/schema.json",
+    "changelog": "@changesets/cli/changelog",
+    "commit": false,
+    "fixed": [["@comet/*"]],
```

Comment:
I'm not sure if I need that but I guess yes?
https://github.com/changesets/changesets/blob/main/docs/config-file-options.md#fixed-array-of-arrays-of-package-names

I guess the comet brevo module should be published like that (with a leading `@`):
 -  comet/brevo-module-api
 -  comet/brevo-module-admin

Reply:
Yes, you'll need that to have same versions for both packages 👍🏼